File: app/src/main/java/com/example/uniDubnaSchedule/MainViewModel.java
package com.example.uniDubnaSchedule;

import android.util.Log;

import androidx.lifecycle.ViewModel;

public class MainViewModel extends ViewModel {

    static {
        Log.e("BRAT", "VM started");
    }

    @Override
    protected void onCleared() {
        super.onCleared();
    }
}

File: app/src/main/java/com/example/udschedule/MainViewModel.kt
package com.example.udschedule

import android.util.Log
import androidx.lifecycle.ViewModel

class MainViewModel : ViewModel() {

    init {
        Log.e("BRAT", "VIEW MODEL STARTED");
    }

    override fun onCleared() {
        Log.e("BRAT", "VIEW MODEL DIED");
        super.onCleared()
    }



}